tests/functional/keystoneapi_controller_test.go

```diff
@@ -342,6 +342,11 @@ var _ = Describe("Keystone controller", func() {
 					},
 				),
 			)
+
+			apiAcc := mariadb.GetMariaDBAccount(keystoneApiName)
+			Expect(apiAcc.Finalizers).To(ContainElement("KeystoneAPI"))
```

Comment:
I'm wondering if we have / need a test case that test the removal of such finalizer.

Reply:
yes, we could. But we might want to do this is a central place in the mariadb-operator, for testing the DeleteFinalizer() function instead of in each operator?

Reply:
I had bugs during creating the bump in placement regarding what name to be used when looking up the db to call db.DeleteFinalizer(). So I think a service operator specific test is worth the effort.

Reply:
ack, I have added a test "removes the finalizers when deleted" which checks before deleting the keystoneapi that the finalizers exist and afterwards that they are gone.